assets/app/view/game/corporation.rb

```diff
@@ -14,20 +14,12 @@ def render
         end
 
         card_style = {
-          display: 'inline-block',
+          border: '1px solid gainsboro',
```

Comment:
should gainsboro and line 22 be current color as opposed to hardcoded to gainsboro and black?

Reply:
For now it’s ok. I’ll revise this later on to fix issue #666 
currentColor only works for font color and would be wrong here. There’s no equivalent for bg – sadly.